### DISCUSSION TITLE
LinkedIn Cookies Remain Set Until Reload

### DISCUSSION THREAD

**Discussion by daleharrison** [orestbida/cookieconsent#569] [Q&A] [answered]
I have `autoclear_cookies` enabled and all the relevant LinkedIn cookies defined in my `cookie_table`.

```
cookie_table: [
  {
    col1: 'AnalyticsSyncHistory'
  },
  {
    col1: 'UserMatchHistory'
  },
  {
    col1: 'li_sugr'
  },
  {
    col1: 'ln_or'
  },
  {
    col1: 'lidc'
  }
]
```

When I accept all cookies, then subsequently reject them, the cookies appear to remain set until I reload the page, at which point they go away.

The domain of the cookies is `linkedin.com`.

Is there a way to have the cookies be deleted immediately without needing to reload the page?

I already tried the following without success:

```
cookie_table: [
  {
    col1: 'AnalyticsSyncHistory',
    domain: 'linkedin.com'
  },
  {
    col1: 'UserMatchHistory',
    domain: 'linkedin.com'
  },
  {
    col1: 'li_sugr',
    domain: 'linkedin.com'
  },
  {
    col1: 'ln_or',
    domain: 'linkedin.com'
  },
  {
    col1: 'lidc',
    domain: 'linkedin.com'
  }
]
```

**Comment by orestbida** [comment, marked as answer]
You can "erase" cookies through javascript: empty their content and make them expire. 

However, it's up to the browser to decide when to fully remove expired cookies. E.g. Firefox immediately removes expired cookies, Chrome does not.
